tests/load-tests/cluster_read_config.yaml

```diff
@@ -99,27 +95,6 @@
   env_variable: {{ var }}
 {% endfor %}
 
-# Cluster nodes info
-- name: metadata.cluster.control-plane.count
-  command: oc get nodes -l node-role.kubernetes.io/master -o name | wc -l
-
-- name: metadata.cluster.control-plane.flavor
-  command: oc get nodes -l node-role.kubernetes.io/master -o json | jq --raw-output '.items | map(.metadata.labels."beta.kubernetes.io/instance-type") | unique | sort | join(",")'
-
-- name: metadata.cluster.control-plane.nodes
-  command: oc get nodes -l node-role.kubernetes.io/master -o json | jq '.items | map(.metadata.name)'
-  output: json
-
-- name: metadata.cluster.compute-nodes.count
-  command: oc get nodes -l node-role.kubernetes.io/worker -o name | wc -l
-
-- name: metadata.cluster.compute-nodes.flavor
-  command: oc get nodes -l node-role.kubernetes.io/worker -o json | jq --raw-output '.items | map(.metadata.labels."beta.kubernetes.io/instance-type") | unique | sort | join(",")'
-
-- name: metadata.cluster.compute-nodes.nodes
-  command: oc get nodes -l node-role.kubernetes.io/worker -o json | jq '.items | map(.metadata.name)'
-  output: json
-
```

Comment:
@naftalysh Why do you remove these metrics?

Reply:
Because with current token, we do not have permissions to execute these commands.

Naftaly, I agree these needs to be fixed, but it does not need to happen in this PR (metrics collection should not fail if some of the commands in the file fails, so it is not critical).

Also I would expect we attempt to find a replacement way for these metrics.

Reply:
@pmacik 
That was the suggestion from Jan as our Prometheus SA doesn't have authorization for that action

Reply:
He also suggested I may ask to elevate the SA privileges but would take longer

Reply:
If you think these are imperative please tell me

Reply:
Naftaly, please keep these as it is and create an issue for us to fix it with error message it is throwing now.

Reply:
@naftalysh Those metrics are useful in CI. If those don't work only in a special case (the staging cluster) they should not be removed generally. That special case should be resolved separately, I agree with @jhutar

Reply:
> @naftalysh Those metrics are useful in CI. If those don't work only in a special case (the staging cluster) they should not be removed generally. That special case should be resolved separately, I agree with @jhutar

I have decided to leave cluster_read_config.yaml with it's previous content (with nodes metrics) to be used for openshift-ci load tests and use another file (cluster_read_config_stage.yaml) for Staging test

Reply:
@pmacik @jhutar 
Could you approve and lgtm?
Last successful run - https://github.com/naftalysh/e2e-tests/actions/runs/6768993231